### PR TITLE
Launch Manage Students Table!

### PR DESCRIPTION
**Pull request by caleybrock**
screen shot 2018-03-05 at 4 52 19 pm

- Remove experiment
- Remove UI for angular table
- Make "move students" button look like a "react style" button, but leave it in angular.

I'll merge this once I get the go ahead from Poorva.

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml [hunk 1]
```diff
@@ -71,82 +71,6 @@ content-type: text/ng-template
...
-        %span{"ng-hide" => "student.confirmDelete"}
-          %button.btn.btn-white{"ng-click" => "edit(student)", "ng-hide" => "student.editing"}= I18n.t('dashboard_action_edit')
-          %button.btn.btn-primary{"ng-click" => "save(student)", "ng-show" => "student.editing", "ng-disabled" => "form.$invalid"}= I18n.t('dashboard_action_save')
-          %button.btn.btn-white{"ng-click" => "cancel(student)", "ng-show" => "student.editing"}= I18n.t('dashboard_action_cancel')
-          %button.btn.btn-danger{"ng-click" => "confirm_delete(student)", "ng-show" => "!student.editing && student.id"}= I18n.t('dashboard_action_remove')
-    %tr{"ng-show" => "teacher_managed_section(section)"}
-      %th{colspan: "4", style: "padding-top:5px;padding-bottom:5px"}
-      %th.manage-th{style: "padding-top:5px;padding-bottom:5px; height:45px"}
-        %button.btn.btn-primary{"ng-click" => "save(section.students)", "ng-show" => "editingAny(section.students)", style: "float:left", "ng-disabled" => "allForm.$invalid"}= I18n.t('dashboard_action_save_all')
-
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml:149]: 🎉!!!!

File: pegasus/sites.v3/code.org/public/teacher-dashboard/nav.haml [hunk 2]
```diff
@@ -50,11 +50,8 @@ content-type: text/ng-template
         -# Just an invisible element for some padding to keep things looking good when we're not showing tabs.
         &nbsp;
 
-  .managebuttons{style: "float:right; margin-bottom: 9px;", "ng-if" => "tab == 'manage' && teacher_managed_section(section)"}
-    %span{"ng-if" => "section.login_type !== 'email'"}
-      %button.btn.btn-white{"ng-click" => "new_student()", "ng-disabled" => "bulk_import.editing"}= I18n.t('dashboard_students_add_student')
-      %button.btn.btn-white{"ng-click" => "bulk_import.editing = true", "ng-disabled" => "bulk_import.editing"}= I18n.t('dashboard_students_add_students')
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/teacher-dashboard/nav.haml:56]: Can we remove the i18n strings `dashboard_students_add_student` and `dashboard_students_add_students`?

File: pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml [hunk 3]
```diff
@@ -71,82 +71,6 @@ content-type: text/ng-template
...
-  %table{style: "width:950px; table-layout:fixed;", 'ng-show' => 'section.students.length > 0', "ng-form" => "allForm"}
-    %colgroup
-      %col{width: "300"}/
-      %col{width: "100"}/
-      %col{width: "100"}/
-      %col{width: "200"}/
-      %col{width: "250"}/
-    %tr
-      %th.manage-th
-        %a{href: "", "ng-click" => "order = 'name'"}= I18n.t('dashboard_students_name')
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml:83]: Likewise any strings in here? (Not sure which are still used elsewhere, there are a lot).

File: pegasus/sites.v3/code.org/public/teacher-dashboard/nav.haml (hunk 2 above)
Comment (caleybrock) [reply pegasus/sites.v3/code.org/public/teacher-dashboard/nav.haml:56]: These are all gsheet controlled, so I'll remove once this PR gets merged.

**Comment by caleybrock** [comment]
Got approval from Poorva.

File: pegasus/sites.v3/code.org/public/teacher-dashboard/nav.haml (hunk 2 above)
Comment (caleybrock) [reply pegasus/sites.v3/code.org/public/teacher-dashboard/nav.haml:56]: Removed all unused strings from gsheet 👍

**Pull request merged**